### DISCUSSION TITLE
Installation triggering problems with Dreambooth?

### DISCUSSION THREAD

**Discussion by taskingoec** [bmaltais/kohya_ss#442] [General]
I have installed kohya as suggested, launching PowerShell as Administrator and overwriting the pip and everything.

After installation I can't use Dreambooth any longer. It just keeps crashing when training new models. Is this possibly related to installing kohya? And if so, how do I set it back to the state before? Or is this unlikely because it is differents venvs?

Many thanks

**Comment by vdruts** [comment]
I don't think that's a good installation strategy and am very surprised the instructions don't include a conda environment creation instead.

**Reply by bmaltais** [reply]
It already use a venv :-) so unless the installations steps were run outside the venv it should not impact anything else.

**Comment by taskingoec** [comment]
I could not find out why my GPU crashed frequently after installing kohya and if it was related. 

However I have found a way that seems to keep my system stable now. I installed EVGA Precision and I slightly underclock my GPU. It seems to help...
